traffic_ops/traffic_ops_golang/servicecategory/servicecategories.go

```diff
@@ -187,3 +187,136 @@ WHERE name=$2 RETURNING last_updated`
 func deleteQuery() string {
 	return `DELETE FROM service_category WHERE name=:name`
 }
+
+// TOServiceCategoryV5 uses tc.ServiceCategoryV5 which has the updated Time Format TimeRFC3339.
+type TOServiceCategoryV5 struct {
+	api.APIInfoImpl `json:"-"`
+	tc.ServiceCategoryV5
+}
+
+func (v *TOServiceCategoryV5) GetLastUpdated() (*time.Time, bool, error) {
+	return api.GetLastUpdatedByName(v.APIInfo().Tx, v.Name, "service_category")
+}
+
+func (v *TOServiceCategoryV5) SetLastUpdated(t tc.TimeRFC3339) { v.LastUpdated = t }
```

Comment:
Time for changed to tc.TimeRFC3339. Rest of the functions are simply copied from the V4 version.